recipes/civetweb/all/conanfile.py

```diff
@@ -64,6 +80,8 @@ def configure(self):
     def requirements(self):
         if self.options.with_ssl:
             self.requires("openssl/1.1.1m")
+        if self.options.with_zlib and tools.Version(self.version) >= "1.15":
```

Comment:
Actually `with_zlib` option should be deleted in `config_options()` if `tools.Version(self.version) < "1.15"`, then you can refer to this option in `requirements` with `self.options.get_safe("with_zlib")`.

Reply:
And I've forgotten: `zlib::zlib` must added to `requires` of at least one component in `package_info()`